src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentForm.java

```diff
@@ -407,15 +408,53 @@ && isInOrgHierarchyNode(getDocument()) && KRADConstants.YES_INDICATOR_VALUE.equa
     }
     
     protected ExtraButton createCreateDVButton() {
-        ExtraButton clearButton = new ExtraButton();
-        clearButton.setExtraButtonProperty("methodToCall.createDV");
-        clearButton.setExtraButtonSource("${" + KFSConstants.EXTERNALIZABLE_IMAGES_URL_KEY + "}buttonsmall_create_DV.gif");
-        clearButton.setExtraButtonAltText("Create DV");
-        clearButton.setExtraButtonOnclick("window.open('" + ConfigContext.getCurrentContextConfig().getProperty(KFSConstants.APPLICATION_URL_KEY)
-                + "/financialDisbursementVoucher.do?methodToCall=createDVFromIWantDoc&docId=" + getDocument().getDocumentNumber()
-                + "');return false;");
-        clearButton.setExtraButtonParams("_blank");
-        return clearButton;
+        ExtraButton creatDVButton = new ExtraButton();
```

Comment:
Misspelling in the local variable name. 
Also, I am not sure the camel casing is correct here "...DV..." but that is what is used elsewhere throughout the file so I do not believe we should change it at this time.

Reply:
updated